src/pages/Lend/LendHeader.vue

```diff
@@ -12,6 +12,9 @@
 			<h1 class="tw-mb-2">
 				Make a loan, <br class="hide-for-medium">change a life
 			</h1>
+
+			<a @click="showPersonalizationWizard()" class="tw-cursor-pointer"> Help me choose the perfect loan.</a>
```

Comment:
could we add a CTA icon here if possible? 😄  check out: 
* [KVMaterialIcon in Storybook](https://608b4cf87f686c00213841b1-ebjvofhmyr.chromatic.com/?path=/docs/kvmaterialicon--default)

and think you can just do like:
```
import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
```

and then
```
<kv-material-icon
			:icon="mdiChevronRight"
/>
```